psamm_import/importer.py

```diff
@@ -121,6 +121,22 @@ def decimal_representer(dumper, data):
     return dumper.represent_scalar('tag:yaml.org,2002:float', value)
 
 
+def detect_extracellular(model):
```

Comment:
Can you add a docstring here documenting the function? Use the docstring from `detect_best_flux_limit()` as a template.